src/framework/components/camera/camera_system.js

```diff
@@ -46,6 +46,11 @@ pc.extend(pc, function () {
             type: "boolean",
             defaultValue: true
         }, {
+            name: "frustumCulling",
+            displayName: "Frustum culling",
+            type: "boolean",
+            defaultValue: false
+        }, {
```

Comment:
Do you think we shouldn't expose it in UI? I think it can be useful to have there.